web/src/main/java/no/nav/foreldrepenger/fordel/web/app/rest/journalføring/OppdaterJournalpostMedTittelDto.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.foreldrepenger.fordel.web.app.rest.journalføring;
+
+import javax.validation.constraints.NotNull;
+
+import java.util.List;
+
+public record OppdaterJournalpostMedTittelDto(String journalpostTittel,
```

Comment:
Her kunne vi kanskje brukt at kodeverk istf String - blir enklere å vedlikeholde senere.

Reply:
Ja, det er nok bedre. Jeg tenker vi kan se på det etter at vi har avgjort titler som skal kunne velges - grunnet løsning for ALT_TITLER som vi snakket om